.changeset/hungry-cougars-tie.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": minor
+---
+
+feat: default `wrangler d1 execute` and `wrangler d1 migrations` commands to local mode first, to match `wrangler dev`
```

Comment:
Let's call this a `refactor`, since it is not really a new feature and having `feat` in a minor release might confuse people.